src/components/App/App.tsx

```diff
@@ -1,17 +1,15 @@
-import { VFC } from "react";
-import { PanelSection, staticClasses } from "decky-frontend-lib";
+import { Fragment, VFC } from "react";
 
 import AvailableSources from "../AvailableSources/AvailableSources";
+import GeneralSoundToggle from "../GeneralSoundToggle/GeneralSoundToggle";
 
 
 const App: VFC = () => {
   return (
-    <PanelSection>
-      <div className={staticClasses.PanelSectionTitle}>
-        {/* Music Mixer */}
-      </div>
+    <Fragment>
```

Comment:
You could use the [Fragment shorthand](https://thisthat.dev/react-fragment-shorthand-vs-longhand) `<></>` like the cool kids 😎

Reply:
Compiler config updated, shorthand used instead